### PR TITLE
Remove closed/make/copy/Copy-jdk.jfr.gmk for JFR Framework

### PR DESCRIPTION
**Pull request by yathamravali**
Both configuration files and metadata.xml are expected in jdk/jfr and modules/jdk.jfr/jdk/jfr/internal/types directory respectively.

https://github.com/ibmruntimes/openj9-openjdk-jdk11/blob/3d65f3fd583bab8ac4a97b914f5aef47c709e70e/src/jdk.jfr/share/classes/jdk/jfr/internal/SecuritySupport.java#L75

https://github.com/ibmruntimes/openj9-openjdk-jdk11/blob/3d65f3fd583bab8ac4a97b914f5aef47c709e70e/src/jdk.jfr/share/classes/jdk/jfr/internal/MetadataHandler.java#L218

However the relevant MakeFile https://github.com/ibmruntimes/openj9-openjdk-jdk11/blob/3d65f3fd583bab8ac4a97b914f5aef47c709e70e/make/copy/Copy-jdk.jfr.gmk is not included. 

Removing `closed/make/copy/Copy-jdk.jfr.gmk` to allow `make/copy/Copy-jdk.jfr.gmk`

Signed-off-by: Ravali Yatham <rayatha1@in.ibm.com>

### CODE REVIEW

**Comment by yathamravali** [comment]
@tajila - Please review

**Comment by tajila** [comment]
We typically start with the https://github.com/ibmruntimes/openj9-openjdk-jdk changes first, then backport to older releases

**Comment by yathamravali** [comment]
@tajila ok, However I don't see this file `closed/make/copy/Copy-jdk.jfr.gmk` in the https://github.com/ibmruntimes/openj9-openjdk-jdk and looks to be JDK11 specific file.

Is there a specific branch that I have to reset to in [openj9-openjdk-jdk](https://github.com/ibmruntimes/openj9-openjdk-jdk) for pushing these changes?

**Comment by keithc-ca** [comment]
> I don't see this file `closed/make/copy/Copy-jdk.jfr.gmk` in the https://github.com/ibmruntimes/openj9-openjdk-jdk and looks to be JDK11 specific file.

In jdk17 and newer versions, such makefiles were reorganized: `make/copy/Copy-jdk.jfr.gmk` was moved to `make/modules/jdk.jfr/Copy.gmk`.
 
> Is there a specific branch that I have to reset to in [openj9-openjdk-jdk](https://github.com/ibmruntimes/openj9-openjdk-jdk) for pushing these changes?

Changes to any jdk version should be based on the "openj9" branch.

However, in later versions, it seems that `*.jfc` files are already handled, and that copying `metadata.xml` is not needed. Here, simply removing `closed/make/copy/Copy-jdk.jfr.gmk` and allowing `make/copy/Copy-jdk.jfr.gmk` to be used should be all that is needed.

File: closed/make/copy/Copy-jdk.jfr.gmk [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # ===========================================================================
-# (c) Copyright IBM Corp. 2018, 2020 All Rights Reserved
+# (c) Copyright IBM Corp. 2023, 2024 All Rights Reserved
```
Comment (keithc-ca) [inline closed/make/copy/Copy-jdk.jfr.gmk:2]: If this file were staying, the first year should not be changed.

**Review by keithc-ca — CHANGES_REQUESTED** [review]

File: closed/make/copy/Copy-jdk.jfr.gmk (hunk 1 above)
Comment (yathamravali) [reply closed/make/copy/Copy-jdk.jfr.gmk]: Thanks Keith, Addressed.

Comment (yathamravali) [reply closed/make/copy/Copy-jdk.jfr.gmk]: Yeah got it. 

> However, in later versions, it seems that *.jfc files are already handled, and that copying metadata.xml is not needed. 

Yes that's correct. The change is specific to JDK11.

> Here, simply removing closed/make/copy/Copy-jdk.jfr.gmk and allowing make/copy/Copy-jdk.jfr.gmk to be used should be all that is needed.

ok, However the  [make/copy/Copy-jdk.jfr.gmk](https://github.com/ibmruntimes/openj9-openjdk-jdk11/blob/3d65f3fd583bab8ac4a97b914f5aef47c709e70e/make/copy/Copy-jdk.jfr.gmk) includes `CopyCommon.gmk` and also I believe it should be wrapped within OPENJ9_ENABLE_JFR check. 

Could you please kindly let me know what is the recommended approach?

Comment (keithc-ca) [reply closed/make/copy/Copy-jdk.jfr.gmk]: The entire `jdk.jfr` module is omitted if `OPENJ9_ENABLE_JFR` is false; see [`closed/custom/common/Modules.gmk`](https://github.com/ibmruntimes/openj9-openjdk-jdk11/blob/openj9/closed/custom/common/Modules.gmk#L46), which means the related makefile fragments (including `make/copy/Copy-jdk.jfr.gmk`) are irrelevant. The only change that is needed is to remove `closed/make/copy/Copy-jdk.jfr.gmk`.

**Comment by keithc-ca** [comment]
Please squash and update the commit message and the description here to reflect the result. Be sure to follow the [commit guidelines](https://github.com/eclipse/omr/blob/master/CONTRIBUTING.md#commit-guidelines).

**Pull request merged**